core/components/molecules/inputMask/InputMask.tsx

```diff
@@ -382,7 +384,7 @@ const InputMask = React.forwardRef<HTMLInputElement, InputMaskProps>((props, for
         required={required}
         onFocus={onFocusHandler}
         onChange={onChangeHandler}
-        onClear={onClearHandler}
+        onClear={!isValueEqualPlaceholder ? onClearHandler : undefined}
```

Comment:
Please move this condition inside the `onClearHandler` method instead of passing undefined